nixos/doc/manual/configuration/profiles/installation-device.xml

```diff
@@ -6,33 +6,31 @@
  <title>Installation Device</title>
 
  <para>
-  Provides a basic configuration for installation devices like CDs. This means
-  enabling hardware scans, using the <link linkend="sec-profile-clone-config">
-  Clone Config profile</link> to guarantee
-  <filename>/etc/nixos/configuration.nix</filename> exists (for
-  <command>nixos-rebuild</command> to work), a copy of the Nixpkgs channel
-  snapshot used to create the install media.
+  Provides a basic configuration for installation devices like CDs.
```

Comment:
At this point `installation device` as a CD even feels antiquated :smile: .
Though I think renaming that is out of scope.

Reply:
Haha yes, the code is also in the `nixos/modules/installer/cd-dvd/` subfolder (and contains `sd-image` and `system-tarball` too) - But I guess cleaning this up should be out of scope for this PR at least.